### DISCUSSION TITLE
What about Workflows commands

### DISCUSSION THREAD

**Discussion by educos35** [pnp/powershell#283] [Q&A] [answered]
Hi, 
I have scripts that works on workflows enumerates, stopping, statuts, etc). After upgrading to this new version of Pnp.PowerShell, it seems that all commands relatives to workflows have been disappeared !?!

How to manage this part of migration ?

Just now, the only solution I found was to retrograde to the old version os PnpSharepointOnline...

Any other solution ?

Regards.

**Comment by erwinvanhunen** [comment, marked as answer]
We've been doing housekeeping in the cmdlets when we started to work on the new module and decided not to port the workflow cmdlets as it's to be deprecated. As we are all volunteers doing this in our free time we have to be careful about resources and time.  We looked at the usages of the various cmdlets and the workflow cmdlets where in the greater picture barely used. Given that the classic workflows have been deprecated and that 2013 workflows are not available by default for new tenants we decided to not port those cmdlets.

Having said that: we are totally okay with it if someone wants to port the old cmdlets to the new modules and create a PR.

The solution right now is indeed to fall back to the legacy module for SharePoint Online. Alternatively one could write the CSOM code in PowerShell, retrieving a clientcontext using Get-PnPContext. But this requires an understanding of CSOM.
